css-values-5/Overview.bs

```diff
@@ -2803,9 +2803,8 @@ Tree Counting Functions: the ''sibling-count()'' and ''sibling-index()'' notatio
 
 	</div>
 
-	Note: Like the rest of CSS (other than [=selectors=]),
-	''sibling-count()'' and ''sibling-index()''
-	operate on the [=flat tree=].
+	Note: These functions may, in the future, have variants that support counting
+	[=flat tree=] siblings.
```

Comment:
```suggestion
	Note: These functions, to match [=selectors=] like '':nth-child()'', operate on the DOM tree,
	rather than the [=flat tree] like most CSS values do.
	They may, in the future, have variants that support counting [=flat tree=] siblings.
```